keystore/src/keystore.rs

```diff
@@ -67,7 +67,7 @@ impl SimpleSecretStore for KeyStore {
     fn insert_account(&self, secret: Secret, password: &Password) -> Result<Address, Error> {
         let keypair = KeyPair::from_private(secret.into()).map_err(|_| Error::CreationFailed)?;
         if self.has_account(&keypair.address())? {
-            Err(Error::AlreadyExists)
+            Err(Error::AlreadyExists(keypair.address()))
```

Comment:
`Address` is not suitable.
The error message is printed to users but user knows only `PlatformAddress` not `Address`.
Please find the way to make `PlatformAddress`.